README.md

```diff
@@ -92,14 +93,18 @@ bin/physlr-make help
     * [networkx](https://networkx.github.io/)
     * [numpy](https://numpy.org/)
     * [scipy](https://www.scipy.org/)
-    * [sklearn](https://scikit-learn.org/stable/)
+    * [sklearn/sickit-learn](https://scikit-learn.org/stable/)
     * [tqdm](https://tqdm.github.io/)	
 
 ### Optional dependencies
 
 - [pigz](https://zlib.net/pigz/) for parallel gzip
 - [zsh](https://sourceforge.net/projects/zsh/) for reporting time and memory usage
 
+### Developers dependencies
+There are additional funcitons in Physlr (specially the python version) for developers to generate granulated reports. These functions may have dependencies some of them listed below:
```

Comment:
```suggestion
There are additional functions in Physlr (especially the python version) for developers to generate more granular reports. The dependencies of these functions are listed below:
```
Feel free to change if this isn't an exhaustive list (although it probably should be?)